src/new-components/banner/banner.stories.mdx

```diff
@@ -0,0 +1,234 @@
+import { Meta, Story, Canvas, ArgsTable, Description } from '@storybook/addon-docs'
+import { Box } from '../box'
+import { Text } from '../text'
+import { Stack } from '../stack'
+import { Banner } from './banner'
+import { ButtonLink } from '../button-link'
+
+<Meta
+    title="Design system/Banner"
+    component={Banner}
+    parameters={{
+        badges: ['accessible'],
+    }}
+/>
+
+export function Icon() {
+    return (
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+            <path
+                fill="currentColor"
+                d="M6 6.653a1 1 0 011.464-.886l10.246 5.37a1 1 0 01-.002 1.773L7.46 18.24a1 1 0 01-1.461-.887V13l6.96-.674a.328.328 0 000-.652L6 11V6.653z"
+            />
+        </svg>
+    )
+}
+
+export function StarIcon() {
+    return (
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+            <g fill="orange">
+                <path
+                    strokeLinejoin="bevel"
+                    d="M8.2 18.6l3.8-2.3 3.8 2.3a.8.8 0 0 0 1-.9l-.9-4.2 3.3-2.8a.8.8 0 0 0-.4-1.3L14.4 9l-1.7-4a.8.8 0 0 0-1.4 0L9.6 9l-4.4.4a.8.8 0 0 0-.4 1.3l3.3 2.8-1 4.2a.8.8 0 0 0 1.1.9z"
+                />
+            </g>
+        </svg>
+    )
+}
+
+export function ArchiveIcon() {
+    return (
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+            <path
+                fill="gray"
+                d="M19.999 9L20 18c0 1.105-.895 2-2 2H6c-1.105 0-2-.895-2-2l-.001-9h16zm-5.49 2H9.491c-.263 0-.491.224-.491.5 0 .268.22.5.491.5h5.018c.263 0 .491-.224.491-.5l-.008-.088c-.041-.229-.242-.412-.483-.412zM3 6c0-1.105.895-2 2-2h14c1.105 0 2 .895 2 2v2H3V6z"
+            />
+        </svg>
+    )
+}
+
+export function EyeIcon() {
+    return (
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24">
+            <g fill="none" fill-rule="evenodd" stroke="gray">
+                <path d="M21.358 12C17.825 7.65 14.692 5.5 12 5.5c-2.624 0-5.67 2.043-9.097 6.181a.5.5 0 0 0 0 .638C6.331 16.457 9.376 18.5 12 18.5c2.692 0 5.825-2.15 9.358-6.5z" />
+                <circle cx="12" cy="12" r="3.5" />
+            </g>
+        </svg>
+    )
+}
+
+export function getButton(buttonText) {
+    return (
+        <ButtonLink variant="primary" tone="destructive">
+            {buttonText}
+        </ButtonLink>
+    )
+}
+
+export function BannerWrapper({ tone, icon, title, description, action }) {
+    return (
+        <Banner tone={tone} icon={icon} title={title} description={description} action={action} />
+    )
+}
```

Comment:
What's the point of this wrapper if all it does is forward the props without doing anything else?